docs/guides/launch.md

```diff
@@ -57,287 +57,83 @@ access it at `http://127.0.0.1:3000/` under credentials `admin/admin`.
 ## (Re)deploy db and contracts
 
 ```
-zk contract redeploy
+zkstack dev contracts
 ```
 
-## Environment configurations
-
-Env config files are held in `etc/env/target/`
-
-List configurations:
-
-```
-zk env
-```
+## Ecosystem Configuration
 
-Switch between configurations:
+Ecosystem configuration can be found in:
 
-```
-zk env <ENV_NAME>
-```
+- `/ZkStack.yaml`
+- `/configs` folder
+- `/chains` folder
 
-Default configuration is `dev.env`, which is generated automatically from `dev.env.example` during `zk init` command
-execution.
+These files are created at ecosystem initialization `zkstack ecosystem init` and at chain initialization
+`zkstack chain init`.
 
 ## Build and run server
 
 Run server:
 
+```bash
+zkstack server
 ```
-zk server
-```
-
-Server is configured using env files in `./etc/env` directory. After the first initialization, file
-`./etc/env/target/dev.env`will be created. By default, this file is copied from the `./etc/env/target/dev.env.example`
-template.
-
-Make sure you have environment variables set right, you can check it by running: `zk env`. You should see `* dev` in
-output.
 
```

Comment:
please give a hint where to find configs

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/3134/commits/9b44cdefbcb4d627a4c25bdbcb5e99ff174ff5e8